src/core/core-impl.hpp

```diff
@@ -66,7 +66,8 @@ class compositor_core_impl_t : public compositor_core_t
     virtual wlr_cursor *get_wlr_cursor() override;
 
     std::string get_xwayland_display() override;
-    pid_t run(std::string command) override;
+    pid_t run(std::string command) override; // Upon success returns the PID of the child process, 0
```

Comment:
The place for docstrings is in the public header, `src/api/wayfire/core.hpp`, not here :)

Reply:
See commit